src/main/java/Duke.java

```diff
@@ -1,10 +1,223 @@
+                break;
+            case TASK_TYPE_EVENT:
+                String eventTask = getTaskFromTaskDescription(taskDescription, EVENT_DATE_INDICATOR);
+                String eventDate = getDateFromTaskDescription(taskDescription, EVENT_DATE_INDICATOR);
+                Event event = new Event(eventTask, eventDate);
+                taskArrayList.add(event);
+                event.printAddTaskMessage();
+                break;
+        }
+        printNumberOfTask();
+    }
+
+    public static void markTaskAsDone(String userInput, ArrayList<Task> taskArrayList) {
+        String[] userInputArray = userInput.split(USER_INPUT_SPLITTER);
+        int doneTaskNumber = Integer.parseInt(userInputArray[1]);
+
+        // We take doneTaskNumber and subtract 1 from it
+        // since ArrayList's index starts from 1
+        doneTaskNumber = doneTaskNumber - 1;
+        Task taskDone = taskArrayList.get(doneTaskNumber);
+        taskDone.setTaskAsDone();
+        taskDone.printMarkTaskAsDone();
+    }
+
+    public static void getAllTask(ArrayList<Task> taskArrayList) {
+        System.out.println("Here are the tasks in your list:");
+        int taskNumber;
+        for (int i = 0; i < taskArrayList.size(); i ++) {
+            Task currentTask = taskArrayList.get(i);
+            taskNumber = i + 1;
+            System.out.println(taskNumber + "." + currentTask.toString());
+        }
+        System.out.println();
+    }
+
+    public static void deleteTaskMessage(Task taskToDelete) {
+        System.out.println("Noted. I've removed this task:\n" +
+                "  " + taskToDelete.toString());
+    }
+
+    public static void deleteTask(String userInput, ArrayList<Task> taskArrayList) {
+        String[] userInputArray = userInput.split(USER_INPUT_SPLITTER);
+        int deleteTaskNumber = Integer.parseInt(userInputArray[1]);
+        int indexOfTaskToDelete = deleteTaskNumber - 1;
+
+        Task taskToDelete = taskArrayList.get(indexOfTaskToDelete);
+        taskArrayList.remove(indexOfTaskToDelete);
+        deleteTaskMessage(taskToDelete);
+        printNumberOfTask();
+    }
+
+    public static void printInvalidTaskNumberProvided() {
+        System.out.println("Task number provided does not exist!\n");
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        welcomeMessage();
+        Scanner scanner = new Scanner(System.in);
+        Pattern doneTaskPattern = Pattern.compile(DONE_TASK_INDICATOR);
+        Pattern deleteTaskPattern = Pattern.compile(DELETE_TASK_INDICATOR);
+        DukeStorage dukeTextFile = new DukeStorage();
+        taskArrayList = dukeTextFile.getTaskArrayList();
+
+        while (true) {
```

Comment:
Be careful with using `true` as a `while` condition which implies that the loop will run infinitely.  

Consider how you can name a [termination condition](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#structure-code-logically) for the while loop and use that instead.